src/hotspot/share/oops/constantPool.cpp

```diff
@@ -2065,7 +2065,7 @@ static void print_cpool_bytes(jint cnt, u1 *bytes) {
     size  += ent_size;
   }
   printf("Cpool size: %d\n", size);
-  fflush(0);
+  fflush(nullptr);
```

Comment:
This looks like a bug. I think someone used 0 aka fd0 when they needed stdout for fflush.

Reply:
I assumed there was some reason for flushing all here, but you are right, this is probably a bug.